library.sh

```diff
@@ -539,7 +539,8 @@ function go_update_deps() {
 
   export GO111MODULE=on
   export GOFLAGS=""
-  export GOSUMDB=off   # Do not use the sum.golang.org service.
+  export GONOSUMDB="${GONOSUMDB},knative.dev/*"
+  export GONOPROXY="${GONOPROXY},knative.dev/*"
```

Comment:
is this the same as direct? Can you link to a doc that describes this behaviour?

Reply:
https://golang.org/cmd/go/#hdr-Module_configuration_for_non_public_modules